week1/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
have you also ran mpm init in the week1 folder? It was not a project folder. Think of what you are trying to achieve. What were you trying to achieve with doing that?